src/types/entities/schelling.ts

```diff
@@ -199,7 +285,7 @@ export class SchellingGame {
 	) {
 		// create the player if it doesn't exist
 		if (!this.players.has(overlay)) {
-			this.players.set(overlay, new Player(overlay, owner, block))
+			this.players.set(overlay, new Player(overlay, owner, block, this.size))
```

Comment:
The whole line thing in Player will be changing when the All Players box gets sorted.